CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.10)
 project(sst-plugininfra VERSION 0.5 LANGUAGES C CXX)
 
 set(CMAKE_CXX_STANDARD 20)
-
+set(SST_PLUGININFRA_BUILD_TESTS true)
```

Comment:
We dont want this in the top level cmake

Reply:
Oops I left that in by accident, apols.